full-stack-tests/backend/src/integration/AzuriteTest.ts

```diff
@@ -53,6 +56,29 @@ export namespace AzuriteTest {
       return blobClient.exists();
     };
 
+    export const overwriteWriteLockValue = async (container: CloudSqlite.CloudContainer, blockName: string) => {
+      const azClient = createAzClient(container.containerId);
+      const blobClient = azClient.getBlockBlobClient(blockName);
+      const tempFilePath = join(KnownLocations.tmpdir, blockName);
+      await blobClient.downloadToFile(tempFilePath);
+
+      const bcv_kv = await open({
```

Comment:
Instead of introducing the dependency on sqlite into full-stack-tests/backend, you should try doing

`const db = new SQLiteDb()`
`db.openDb("Path-to-db", ...)`

Reply:
I would also maybe rename the function to be setWriteLockToExpired? Just want to capture that the function is setting it to a time in the past.

Reply:
Resolved in [commit c79429aefcf07d9be974cdf095add0ce657c9c55](https://github.com/iTwin/itwinjs-core/pull/7000/commits/c79429aefcf07d9be974cdf095add0ce657c9c55)